tests/systemd_testsuite/build_from_upstream_git.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: fetch sources from upstream git and build systemd inclusive
+#          testsuite
+# Maintainer: Thomas Blume <tblume@suse.com>
+
+
+use base "opensusebasetest";
+
+use strict;
+use warnings;
+use known_bugs;
+use testapi;
+use power_action_utils 'power_action';
+use utils 'zypper_call';
+use version_utils qw(is_opensuse is_sle is_tumbleweed);
+
+
+sub installandbuild {
+    zypper_call 'in osc';
+    assert_script_run "osc branch openSUSE:Factory systemd";
```

Comment:
you want to build locally, I hope?.

Reply:
Actually that is a preparation for my planned testservice for systemd upstream, which will be on a dedicated local machine.
It isn't used in the tumbleweed test.
Should I remove build_from_upstream_git.pm  from this submission or can I leave it to continue this later on?

Reply:
Ok, I've removed the build_from_upstream_git.pm file. It doesn't belong to the tumbleweed tests and only causes confusion.

Reply:
I think if it's not scheduled, there's a script that will complain about the test not being scheduled :P